### ISSUE TITLE
Move the background tasks list from the admin UI to the primary UI

### ISSUE THREAD

**Issue by jeremystretch** [netbox-community/netbox#14729]
### NetBox version

v3.7.0

### Feature type

Change to existing functionality

### Proposed functionality

NetBox currently lists all background tasks under the admin UI (`/admin/background-tasks/`), in a view provided by the `django-rq` package. Replicate this view in the primary UI, under the "admin" menu section.

### Use case

Users will be able to inspect active and completed background tasks without leaving the primary UI, and unblocks #12325.

### Database changes

None

### External dependencies

None

**Comment by tagur87** [comment]
@jeremystretch - I recently found an issue with viewing background tasks from the admin UI using redis sentinel according to these instructions: https://demo.netbox.dev/static/docs/configuration/required-settings/#using-redis-sentinel

I get the following error when browsing to /admin/background-tasks/:

```
<class 'KeyError'>

'host'

Python version: 3.11.4
NetBox version: 3.7.0
Plugins: 
  netbox_ip_reservations: 0.5.3
```

Not sure if I should create a separate issue, since this is being reworked as part of the removal of the Admin UI?

Thanks!

**Comment by jeremystretch** [comment]
@tagur87 please open a separate bug report including instructions to reproduce. Thanks!

**Comment by tagur87** [comment]
> @tagur87 please open a separate bug report including instructions to reproduce. Thanks!

https://github.com/netbox-community/netbox/issues/14752
